### ISSUE TITLE
base_module.py AttributeError with top_modules config

### ISSUE THREAD

**Issue by plijnzaad** [ewels/MultiQC#1274]
**Description of bug:**
I tried to run multiqc on a FastQC-produced zip file, as follows:

```bash
multiqc --config config.yaml --force --title foo --zip-data-dir --filename foo-QC.html \
    --outdir . -m star -m fastqc \
    ./call-fastqQC/execution/TM901-compTrim.u_fastqc.zip \
    ./call-fastqQC/execution/TM901-untrimmed.u_fastqc.zip \
    ./call-fastqQC/execution/TM901-adaptTrim.u_fastqc.zip
```
The zips are directly from FastQC; my impression was that this is enough?

The config I used was:

```yaml
---

sp:
  fastqc/zip:
    fn: '*u_fastqc.zip'

top_modules:
  - fastqc:
```

(also attached)
[config.yaml.txt](https://github.com/ewels/MultiQC/files/5096507/config.yaml.txt)

**MultiQC Error log:**

```console
[INFO   ]         multiqc : This is MultiQC v1.9
[INFO   ]         multiqc : Template    : default
[INFO   ]         multiqc : Report title: foo
[INFO   ]         multiqc : Searching   : call-fastqQC/execution/TM901-compTrim.u_fastqc.zip
[INFO   ]         multiqc : Searching   : call-fastqQC/execution/TM901-untrimmed.u_fastqc.zip
[INFO   ]         multiqc : Searching   : call-fastqQC/execution/TM901-adaptTrim.u_fastqc.zip
[INFO   ]         multiqc : Only using modules star, fastqc
[ERROR  ]         multiqc : Oops! The 'fastqc' MultiQC module broke... 
  Please copy the following traceback and report it at https://github.com/ewels/MultiQC/issues 
  If possible, please include a log file that triggers the error - the last file found was:
    None
============================================================
Module fastqc raised an exception: Traceback (most recent call last):
  File "/hpc/local/CentOS7/gen/software/multiqc-1.9/venv/lib/python3.6/site-packages/multiqc/multiqc.py", line 569, in run
    output = mod()
  File "/hpc/local/CentOS7/gen/software/multiqc-1.9/venv/lib/python3.6/site-packages/multiqc/modules/fastqc/fastqc.py", line 37, in __init__
    info="is a quality control tool for high throughput sequence data,"\
  File "/hpc/local/CentOS7/gen/software/multiqc-1.9/venv/lib/python3.6/site-packages/multiqc/modules/base_module.py", line 26, in __init__
    self.name = mod_cust_config.get('name', name)
AttributeError: 'NoneType' object has no attribute 'get'
============================================================
[WARNING]         multiqc : No analysis results found. Cleaning up..
[INFO   ]         multiqc : MultiQC complete

```

**File that triggers the error:**

[TM901-compTrim.u_fastqc.zip](https://github.com/ewels/MultiQC/files/5096492/TM901-compTrim.u_fastqc.zip)

**MultiQC run details (please complete the following):**

* Command used to run MultiQC: `multiqc --config config.yaml --force --title foo --zip-data-dir --filename foo-QC.html --outdir . -m star -m fastqc ./call-fastqQC/execution/TM901-compTrim.u_fastqc.zip ./call-fastqQC/execution/TM901-untrimmed.u_fastqc.zip ./call-fastqQC/execution/TM901-adaptTrim.u_fastqc.zip`

* MultiQC Version: 1.9
* Operating System: CentOS Linux release 7.7.1908 (Core)
* Python Version: 3.6.1
* Method of MultiQC installation: probably conda

**Additional context**

**Comment by ewels** [comment]
Hi @plijnzaad,

Thanks for this! Sorry for the slow response, I'll get to looking / fixing this when I can.

On first glance my guess is that it's because you're doing `- fastqc:` instead of just `- fastqc` in the YAML (making it an empty dict instead of a string). But that's just a guess, and MultiQC should be able to handle this case if true.

Phil

**Comment by fgvieira** [comment]
I am getting the same error for `featureCounts`, `biobambam2`, `picard` and `fastqc`, both woth MultiQC v1.9 and v1.10

For example:
```
[2021-03-09 22:47:52,016] multiqc                                            [ERROR  ]  Oops! The 'biobambam2' MultiQC module broke...
  Please copy the following traceback and report it at https://github.com/ewels/MultiQC/issues
  If possible, please include a log file that triggers the error - the last file found was:
    None
============================================================
Module biobambam2 raised an exception: Traceback (most recent call last):
  File "cache/conda/8303e0a62ce0a269e11e22607186aa0d/lib/python3.6/site-packages/multiqc/multiqc.py", line 569, in run
    output = mod()
  File "cache/conda/8303e0a62ce0a269e11e22607186aa0d/lib/python3.6/site-packages/multiqc/modules/biobambam2/biobambam2.py", line 23, in __init__
    info="provides tools for early stage alignment file processing")
  File "cache/conda/8303e0a62ce0a269e11e22607186aa0d/lib/python3.6/site-packages/multiqc/modules/base_module.py", line 26, in __init__
    self.name = mod_cust_config.get('name', name)
AttributeError: 'NoneType' object has no attribute 'get'
```

**Comment by ewels** [comment]
Tested and confirmed that changing

```yaml
top_modules:
  - fastqc:
```

to this:

```yaml
top_modules:
  - fastqc
```

..resolves the issue. Will take a look now to see if I can make it handle the incorrect YAML format.

**Comment by ewels** [comment]
Fixed in v1.11dev so both styles of YAML config should work. Before then, just remove the superfluous `:` and you should be fine 👍🏻 

Thanks for reporting!

Phil
